inference-engine/cmake/features_ie.cmake

```diff
@@ -74,13 +74,13 @@ ie_dependent_option (GAPI_TEST_PERF "if GAPI unit tests should examine performan
 
 ie_dependent_option (ENABLE_MYRIAD_MVNC_TESTS "functional and behavior tests for mvnc api" OFF "ENABLE_TESTS;ENABLE_MYRIAD" OFF)
 
-ie_dependent_option (ENABLE_DATA "fetch models from testdata repo" ON "ENABLE_FUNCTIONAL_TESTS;NOT ANDROID" OFF)
+ie_dependent_option (ENABLE_DATA "fetch models from testdata repo" ON "ENABLE_FUNCTIONAL_TESTS;NOT ANDROID" ON)
```

Comment:
Should it be ie_option instead? Option dependency looks void since it has ON on both ends of the ternary operator.

`ie_option (ENABLE_DATA "fetch models from testdata repo" ON)`

Reply:
@asomsiko I reverted these changes.